README.md

```diff
@@ -1,17 +1,40 @@
-# Cucumber JVM
-
-[![#StandWithUkraine](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/badges/StandWithUkraine.svg)](https://vshymanskyy.github.io/StandWithUkraine)
-[![OpenCollective](https://opencollective.com/cucumber/backers/badge.svg)](https://opencollective.com/cucumber)
-[![OpenCollective](https://opencollective.com/cucumber/sponsors/badge.svg)](https://opencollective.com/cucumber)
-[![Maven Central](https://img.shields.io/maven-central/v/io.cucumber/cucumber-java.svg?label=Maven%20Central)](https://central.sonatype.com/artifact/io.cucumber/cucumber-java)
-[![Build Status](https://github.com/cucumber/cucumber-jvm/workflows/Cucumber%20CI/badge.svg)](https://github.com/cucumber/cucumber-jvm/actions)
-[![Coverage Status](https://codecov.io/gh/cucumber/cucumber-jvm/branch/main/graph/badge.svg)](https://codecov.io/gh/cucumber/cucumber-jvm/branch/main)
-
-Cucumber-JVM is a pure Java implementation of Cucumber.
-You can [run](https://cucumber.io/docs/cucumber/api/#running-cucumber) it with
-the tool of your choice.
-
-Cucumber-JVM also integrates with all the popular
+<h1 align="center">
+  <img src="https://raw.githubusercontent.com/cucumber/cucumber-js/46a5a78107be27e99c6e044c69b6e8f885ce456c/docs/images/logo.svg" alt="Cucumber logo" width="75">
+  <br>
+  Cucumber JVM
+</h1>
+<p align="center">
+  <b>Automated tests in plain language, for JVM</b>
+</p>
+
+<p align="center">
+  <a href="https://central.sonatype.com/artifact/io.cucumber/cucumber-java" style="text-decoration: none">
+    <img src="https://img.shields.io/maven-central/v/io.cucumber/cucumber-java?style=flat&color=dark-green&label=Maven%20Central" alt="Cucumber JVM version">
+  </a>
+  <a href="https://github.com/cucumber/cucumber-jvm/actions" style="text-decoration: none">
```

Comment:
It doesn't matter too much, but is there a reason for not using mark down urls here?

Reply:
When applying with other repos a year back, GitHub wasn't rendering the alignment correctly. However re-evaluating based on your question, its working - so have reverted to conventional, leaner markdown format. Thanks for checking!